bundles/org.eclipse.jface.text/src/org/eclipse/jface/internal/text/html/BrowserInformationControl.java

```diff
@@ -585,6 +585,13 @@ public BrowserInformationControlInput getInput() {
 		return fInput;
 	}
 
+	/**
+	 * @return true if the location is of the browser's custom text page, otherwise false
+	 */
+	public Boolean isLocationForBrowserCustomText(String loc) {
```

Comment:
Is there a reason for using the boxed type instead of primite `boolean`? If not, please change to use boolean.
And please use a speaking parameter name, e.g., `location`.

Reply:
Nope it was a mistake. Fixed it

Reply:
```suggestion
	public boolean isCustomTextPage(String loc) {
```

The old name is utterly confusing without the javadoc and the discussions here I won't be able to even guess what this method is supposed for, maybe even rename for SWT as well?

Beside that, why can't one get acces to the fbrowser and ask it directly?

Reply:
> The old name is utterly confusing without the javadoc and the discussions here I won't be able to even guess what this method is supposed for, maybe even rename for SWT as well?

I agree that the name needs to be changed (https://github.com/eclipse-platform/eclipse.platform.ui/pull/2211#discussion_r1746601006)
 
> Beside that, why can't one get acces to the fbrowser and ask it directly?

That would break encapsulation. The browser is, by intent, properly hidden as an implementation detail of the `BrowserInformationControl`. Allowing a user to access leads (1) to worse design (because of less encapsulation, more necessity to know about the internals to properly use it) and (2) introduces the risk of problems as the current implementation assumes that the browser is hidden and cannot change its state due to access by anyone else.